include/mega/filesystem.h

```diff
@@ -172,6 +172,18 @@ struct MEGA_API InputStreamAccess
     virtual ~InputStreamAccess() { }
 };
 
+class MEGA_API FileInputStream : public InputStreamAccess
+{
+    FileAccess *fileAccess;
+    m_off_t offset;
+
+public:
+    FileInputStream(FileAccess *fileAccess);
+    virtual m_off_t size();
+    virtual bool read(byte *buffer, unsigned size);
+    virtual ~FileInputStream();
```

Comment:
Since we are moving this class, we could update the interface to use the 'override' keyword rather than `virtual`, it was written pre-c++11.   Since the destructor is empty I think we can remove that too?  thanks